website/content/en/status/report-2024-07-2024-09/simd.adoc

```diff
@@ -0,0 +1,26 @@
+=== SIMD enhancements for aarch64
+
+Links: +
+link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[EuroBSDCon 2024 presentation] URL: link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[] +
```

Comment:
In addition to that, could you provide a link to the slides or presentation notes, for accessibility?